.travis.yml

```diff
@@ -46,7 +47,7 @@ before_install:
   galaxy_kickstart`
 - docker ps
 install:
-- sudo su $GALAXY_TRAVIS_USER -c 'pip install --user bioblend[testing]'
+  - sudo su $GALAXY_TRAVIS_USER -c 'pip install --user https://github.com/mvdbeek/bioblend/archive/test_data.zip pytest'
```

Comment:
Ah ! Indeed because I though this way avoids to put pytest in the pip install command. But I may be mixed...